interface/practice/ins_search.php

```diff
@@ -168,13 +168,13 @@ function validate(f) {
    $ins_id = generate_id();
 
    sqlInsert("INSERT INTO insurance_companies ( " .
-    "id, name, attn, cms_id, freeb_type, x12_receiver_id, x12_default_partner_id " .
+    "id, name, attn, cms_id, ins_co_type, x12_receiver_id, x12_default_partner_id " .
```

Comment:
The bug noted stems from this script where you should be using ins_code_type as the base for all the text changes here rather than ins_co_type .